src/fsharp/FSharp.Compiler.Service/FSharp.Compiler.Service.fsproj

```diff
@@ -354,288 +284,270 @@
     </Compile>
-    <Compile Include="..\vs\ServiceLexing.fs">
+    <Compile Include="../vs/ServiceLexing.fs">
       <Link>Service/ServiceLexing.fs</Link>
     </Compile>
-    <Compile Include="..\vs\ServiceParseTreeWalk.fs">
+    <Compile Include="../vs/ServiceParseTreeWalk.fs">
       <Link>Service/ServiceParseTreeWalk.fs</Link>
     </Compile>
-    <Compile Include="..\vs\ServiceNavigation.fsi">
+    <Compile Include="../vs/ServiceNavigation.fsi">
       <Link>Service/ServiceNavigation.fsi</Link>
     </Compile>
-    <Compile Include="..\vs\ServiceNavigation.fs">
+    <Compile Include="../vs/ServiceNavigation.fs">
       <Link>Service/ServiceNavigation.fs</Link>
     </Compile>
-    <Compile Include="..\vs\ServiceParamInfoLocations.fsi">
+    <Compile Include="../vs/ServiceParamInfoLocations.fsi">
       <Link>Service/ServiceParamInfoLocations.fsi</Link>
     </Compile>
-    <Compile Include="..\vs\ServiceParamInfoLocations.fs">
+    <Compile Include="../vs/ServiceParamInfoLocations.fs">
       <Link>Service/ServiceParamInfoLocations.fs</Link>
     </Compile>
-    <Compile Include="..\vs\ServiceUntypedParse.fsi">
+    <Compile Include="../vs/ServiceUntypedParse.fsi">
       <Link>Service/ServiceUntypedParse.fsi</Link>
     </Compile>
-    <Compile Include="..\vs\ServiceUntypedParse.fs">
+    <Compile Include="../vs/ServiceUntypedParse.fs">
       <Link>Service/ServiceUntypedParse.fs</Link>
     </Compile>
-    <Compile Include="..\..\utils\reshapedmsbuild.fs">
+    <Compile Include="../../utils/reshapedmsbuild.fs">
       <Link>Service/reshapedmsbuild.fs</Link>
     </Compile>
-    <Compile Include="..\SimulatedMSBuildReferenceResolver.fs">
+    <Compile Include="../SimulatedMSBuildReferenceResolver.fs">
       <Link>Service/SimulatedMSBuildReferenceResolver.fs</Link>
     </Compile>
-    <!--
-    <Compile Include="..\vs\ServiceAssemblyContent.fs">
-      <Link>Service/ServiceAssemblyContent.fs</Link>
-    </Compile>
-    <Compile Include="..\vs\ServiceXmlDocParser.fs">
-      <Link>Service/ServiceXmlDocParser.fs</Link>
-    </Compile>
--->
-    <Compile Include="..\vs\service.fsi">
+    <Compile Include="../vs/service.fsi">
       <Link>Service/service.fsi</Link>
     </Compile>
-    <Compile Include="..\vs\service.fs">
+    <Compile Include="../vs/service.fs">
       <Link>Service/service.fs</Link>
     </Compile>
-    <Compile Include="..\vs\SimpleServices.fsi">
+    <Compile Include="../vs/SimpleServices.fsi">
       <Link>Service/SimpleServices.fsi</Link>
     </Compile>
-    <Compile Include="..\vs\SimpleServices.fs">
+    <Compile Include="../vs/SimpleServices.fs">
       <Link>Service/SimpleServices.fs</Link>
     </Compile>
-    <Compile Include="..\fsi\fsi.fsi">
+    <Compile Include="../fsi/fsi.fsi">
       <Link>Service/fsi.fsi</Link>
     </Compile>
-    <Compile Include="..\fsi\fsi.fs">
+    <Compile Include="../fsi/fsi.fs">
       <Link>Service/fsi.fs</Link>
     </Compile>
+    <Compile Include="illex.fs" />
```

Comment:
Probably just some of my mad ravings trying to get it to build :(  I'll definitely revert changes to the non-.NET core project files